core/App/utils/helpers.ts

```diff
@@ -12,6 +12,21 @@ import { parseUrl } from 'query-string'
 
 import { Attribute, Predicate } from '../types/record'
 
+export function parseCredDefName(credDefId?: string): string {
```

Comment:
Can we organize/group functions by the info they extract? Ex. schema functions grouped with other schema functions, and likewise with cred defs.